src/qs8-gemm/MRx4c16-wasmdot.c.in

```diff
@@ -7,19 +7,21 @@ $assert DATATYPE in ["QC8", "QD8", "QS8"]
 $assert DATATYPE == "QD8" or REQUANTIZATION == "FP32"
 $assert DATATYPE != "QD8" or not REQUANTIZATION
 $assert MR <= 4
+$assert NR == 4 or NR == 8
+$assert (SDOT == 0 and DATATYPE == "QC8") or SDOT == 1
 #include <assert.h>
 
 #include <wasm_simd128.h>
 
-#include "xnnpack/gemm.h"
-#include "xnnpack/math.h"
-
+#include <xnnpack/gemm.h>
```

Comment:
please change to #include "xnnpack/gemm.h" for xnn headers

Reply:
Thanks for the comments! I updated them in the latest change.